spec/models/band_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+describe Band do
+  describe 'associations' do
+    it { should have_many(:musicians) }
```

Comment:
Why do you use `should` syntax in all specs? You should use the new syntax: `is_expected`.